storage/localstore_test.go

```diff
@@ -7,22 +7,25 @@ import (
 	"path/filepath"
```

Comment:
if `localstore.go` was deleted, can we rename this file?

Reply:
no this is fine, it is still testing localstore